### ISSUE TITLE
No longer works with Laravel 7.x

### ISSUE THREAD

**Issue by alistairreynolds** [TheoKouzelis/laravel-airbrake#12]
After updating to 7.x, composer says there's a conflict

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: remove kouz/laravel-airbrake 0.5.0
    - Conclusion: don't install laravel/framework v7.1.3
```

**Comment by TheoKouzelis** [comment]
Hello @alistairreynolds 

It looks like Laravel 7.x requires Monolog 2.x https://github.com/laravel/framework/blob/7.x/composer.json#L27

And Monolog 2.x is currently in compatible with  PHPBrake 0.7.2 https://github.com/airbrake/phpbrake/blob/v0.7.2/composer.json#L15

There is an open issue on the PHPBrake repo https://github.com/airbrake/phpbrake/issues/97. Until that is resolved this package won't be able to support Laravel 7..x

**Comment by TheoKouzelis** [comment]
PR pending here https://github.com/airbrake/phpbrake/pull/101

**Comment by alistairreynolds** [comment]
Good timing, thanks for the info!

**Comment by Nolanus** [comment]
PR is merged and released with phpbrake v0.7.3

**Comment by TheoKouzelis** [comment]
Fixed in release 0.6.0 https://github.com/TheoKouzelis/laravel-airbrake/releases/tag/0.6.0
